Default gateway.auto_import_dangling_indices to false (#59302)
elastic/elasticsearch: docs/reference/release-notes/7.9.asciidoc

  default that's used for `indices.recovery.max_bytes_per_sec`. This means
  that no behavioral change will be observed by clusters where the recovery
  and restore settings had not been adapted from the defaults. {es-pull}58658[#58658]

> Dangling indices::
> * A new <<dangling-indices-api,Dangling indices API>> is now available. You
>   can use it to list, import or delete
>   <<dangling-indices,dangling indices>>. Because of this, automatically
>   importing dangling indices is now disabled by default. There are several
>   issues with this behavior, and you should use the dedicated API instead.
>   If you need to re-enable auto-imports, set
>   `gateway.auto_import_dangling_indices` to `true`.  Auto-imports are now
>   deprecated and will be removed in 8.0 {es-pull}58176[#58176] {es-pull}58898[#58898] (issue: {es-issue}48366[#48366])

Reviewer: ```suggestion
Dangling indices::
* A new <<dangling-indices-api,Dangling indices API>> is now available, which you
  can use to list, import, or delete
  <<dangling-indices,dangling indices>>. Because importing dangling indices
  into the cluster using `gateway.auto_import_dangling_indices` is unsafe, use the dedicated API instead.
  
  Automatically importing dangling indices is now deprecated, disabled by default, and will be removed in (es) 8.0. If you need to re-enable auto-imports, set
  `gateway.auto_import_dangling_indices` to `true`
   {es-pull}58176[#58176] {es-pull}58898[#58898] (issue: {es-issue}48366[#48366])
```

Reviewer: >Because of this, automatically
  importing dangling indices is now disabled by default.

I think that we should clarify this language to explain what "this" is. I reused some of the language from [58751](https://github.com/elastic/elasticsearch/pull/58751/files#diff-d5961dc535d48b0d23e553177bc81b2fR61-R62), which emphasizes that using `gateway.auto_import_dangling_indices` is unsafe. Please let me know if I misconstrued the original meaning of this explanation.

  default that's used for `indices.recovery.max_bytes_per_sec`. This means
  that no behavioral change will be observed by clusters where the recovery
  and restore settings had not been adapted from the defaults. {es-pull}58658[#58658]

Dangling indices::
* Automatically importing dangling indices is now deprecated, disabled by

>   default, and will be removed in (es) 8.0. See the

Reviewer 2: I think you meant this:

```suggestion
  default, and will be removed in {es} 8.0. See the
```
